### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md [hunk 1]
```diff
@@ -16,19 +16,16 @@ Thank you for letting us know you are experiencing a similar situation. Due to t
 
 Thank you for providing feedback, and we're sorry to hear it's not behaving as you expect. Based on your description so far, it sounds like you are experiencing a problem which is historically hard to diagnose and resolve using the normal "steps to reproduce". We created a set of instructions for providing additional information which will help us track down the true source of the problems.
 
-Based on the information provided so far, the most likely scenario to follow is for "Crashes". Please take a look at the following document to provide the feedback most relevant for the problems you would like to see fixed:
-https://docs.microsoft.com/en-us/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#crashes
+Based on the information provided so far, the most likely scenario to follow is for "Crashes". To provide the feedback most relevant for the problems you would like to see fixed, see [Crashes](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#crashes).
```
Comment (Youssef1313) [inline docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md:19]:
```suggestion
Based on the information provided so far, the most likely scenario to follow is for "Crashes". To provide the feedback most relevant for the problems you would like to see fixed, see [Crashes](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed#crashes).
```

File: docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md [hunk 2]
```diff
@@ -16,19 +16,16 @@ Thank you for letting us know you are experiencing a similar situation. Due to t
...
 
 Thank you for providing feedback, and we're sorry to hear it's not behaving as you expect. Based on your description so far, it sounds like you are experiencing a problem which is historically hard to diagnose and resolve using the normal "steps to reproduce". We created a set of instructions for providing additional information which will help us track down the true source of the problems.
 
-Based on the information provided so far, the most likely scenario to follow is for "Slowness and High CPU Issues". Please look at the following document to provide the feedback most relevant for the problems you would like to see fixed:
-https://docs.microsoft.com/en-us/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#slowness-and-high-cpu-issues
+Based on the information provided so far, the most likely scenario to follow is for "Slowness and High CPU Issues". To provide the feedback most relevant for the problems you would like to see fixed, see [Slowness and High CPU Issues](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#slowness-and-high-cpu-issues).
```
Comment (Youssef1313) [inline docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md:25]:
```suggestion
Based on the information provided so far, the most likely scenario to follow is for "Slowness and High CPU Issues". To provide the feedback most relevant for the problems you would like to see fixed, see [Slowness and High CPU Issues](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed#slowness-and-high-cpu-issues).
```

File: docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md [hunk 3]
```diff
@@ -16,19 +16,16 @@ Thank you for letting us know you are experiencing a similar situation. Due to t
...
 
 ### Out-Of-Process Issues
 
 Thank you for providing feedback, and we're sorry to hear it's not behaving as you expect. Based on your description so far, it sounds like you are experiencing a problem which is historically hard to diagnose and resolve using the normal "steps to reproduce". We created a set of instructions for providing additional information which will help us track down the true source of the problems.
 
-Based on the information provided so far, the most likely scenario to follow is for "Out-Of-Process Issues". Please look at the following document to provide the feedback most relevant for the problems you would like to see fixed:
-https://docs.microsoft.com/en-us/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#out-of-process-issues
+Based on the information provided so far, the most likely scenario to follow is for "Out-Of-Process Issues". To provide the feedback most relevant for the problems you would like to see fixed, see [Out-of-process issues](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed?view=vs-2019#out-of-process-issues).
```
Comment (Youssef1313) [inline docs/wiki/Responding-to-customers-encountering-crashes-or-performance-issues.md:31]:
```suggestion
Based on the information provided so far, the most likely scenario to follow is for "Out-Of-Process Issues". To provide the feedback most relevant for the problems you would like to see fixed, see [Out-of-process issues](https://docs.microsoft.com/visualstudio/ide/how-to-increase-chances-of-performance-issue-being-fixed#out-of-process-issues).
```

**Comment by JustPies** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@BillWagner @333fred

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by 333fred** [comment]
@alexbuckgit these changes generally seem ok, but I'm confused as to the purpose. AFAIK, none of these documents are published on docs anywhere. Is that not the case?

**Comment by BillWagner** [comment]
> these changes generally seem ok, but I'm confused as to the purpose. AFAIK, none of these documents are published on docs anywhere. Is that not the case?

@333fred Correct. These aren't published. The only changes in this PR remove the locale from the target URL. That's good because the docs site will redirect to the correct locale for the readers' browser.

**Comment by 333fred** [comment]
Sure, I'm just curious as to why these changes were generated at all, given that the docs aren't published anywhere.

**Review by 333fred — APPROVED** [review]

**Pull request merged**